src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Confirm.tsx

```diff
@@ -185,24 +185,15 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     quote,
   ])
 
-  const memo = useMemo(() => {
-    // Note: this is *always* an UTF-8 string as we use this for the memo field of depositWithExpiry()
-    // The only time this would need to be converted to hex is if we were to use it for EVM sends,
-    // which we should never do since depositWithExpiry() is the way
-    const memoUtf8 = quote?.memo
-    if (!memoUtf8) return
-    return memoUtf8
-  }, [quote?.memo])
-
```

Comment:
@kaladinlight the main intent here was the comment - agree we don't need the memoized expression but can we at least keep `const memoUtf8 = quote?.memo` - and perhaps a comment, to signify that in this specific case we should never `toHex` it?